setup.py

```diff
@@ -276,13 +276,6 @@ def get_long_description():
         Operating System :: POSIX :: Linux
         Programming Language :: Python
         Programming Language :: Python :: 3
-        Programming Language :: Python :: 3.6
```

Comment:
Removing these classifiers - they're a pain to maintain and provide little value. Note that we do have a `python_requires` keyword below giving the minimum supported version of Python.